gp/cgp_graph.py

```diff
@@ -179,8 +179,8 @@ def __getitem__(self, key):
     def to_str(self):
 
         self._format_output_str_of_all_nodes()
-
-        return '[{}]'.format(', '.join(node.output_str for node in self.output_nodes))
+        out_str = ', '.join(node.output_str for node in self.output_nodes)
+        return f'[{out_str}]'
```

Comment:
would this work?
```suggestion
        return f'[{", ".join(node.output_str for node in self.output_nodes)}]'
```

Reply:
Yes, but I found it kind of ugly and unreadble, too many brackets and quotation marks. But if you prefer this, it's fine for me.

Reply:
ok, let's keep it your way. we're writing for humans after all :)